### PR TITLE
Fix Next-Page url if service has key in url (fixes #157) 	

### LINKED ISSUE #157: KeyError with Kinto collection_id
TravisCI fails because of some route_url error with Kinto : 

```
 File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/cliquet-1.4.1-py2.7.egg/cliquet/resource.py", line 251, in get_records

next_page = self._next_page_url(sorting, limit, records[-1])
 File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/cliquet-1.4.1-py2.7.egg/cliquet/resource.py", line 655, in _next_page_url

next_page_url = self.request.route_url(service.name, _query=params)

File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/pyramid-1.5.4-py2.7.egg/pyramid/url.py", line 279, in route_url

path = route.generate(kw) # raises KeyError if generate fails

File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/pyramid-1.5.4-py2.7.egg/pyramid/urldispatch.py", line 239, in generator

result = gen % newdict # native string result

KeyError: 'collection_id'; resource_name=test; uid=basicauth_206691a25679e4e1135f16aa77ebcf211c767393c4306cfffe6cc228ac0886b6; errno=None; agent=python-requests/2.6.0 CPython/2.7.9 
```

### PR DESCRIPTION
**Pull request by leplatrem**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -37,7 +37,7 @@ need-kinto-running:
 	@curl http://localhost:8888/v0/ 2>/dev/null 1>&2 || (echo "Run 'make runkinto' before starting tests." && exit 1)
 
 runkinto:
-	$(VENV)/bin/pserve cliquet/tests/config/kinto.ini
+	$(VENV)/bin/pserve cliquet/tests/config/kinto.ini --reload
```
Comment (ametaireau) [inline Makefile]: cool!

File: cliquet/resource.py [hunk 2]
```diff
@@ -652,7 +652,8 @@ def _next_page_url(self, sorting, limit, last_record):
         params['_token'] = token
 
         service = current_service(self.request)
-        next_page_url = self.request.route_url(service.name, _query=params)
+        next_page_url = self.request.route_url(service.name, _query=params,
+                                               **self.request.matchdict)
```
Comment (ametaireau) [inline cliquet/resource.py]: Seems to miss tests ?

Comment (leplatrem) [inline cliquet/resource.py]: I really don't know how to unit test this :( Any hints ?

Before : tests broken. After : tests green.

**Comment by ametaireau** [comment]
Oh, if tests were broken before then we're all good.

**Pull request merged**